.github/workflows/rust.yml

```diff
@@ -0,0 +1,47 @@
+name: Rust
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Install sccache
+      run: |
+        wget https://github.com/mozilla/sccache/releases/download/v0.5.3/sccache-v0.5.3-x86_64-unknown-linux-musl.tar.gz
+        tar -xzf sccache-v0.5.3-x86_64-unknown-linux-musl.tar.gz
+        sudo mv sccache-v0.5.3-x86_64-unknown-linux-musl/sccache /usr/local/bin/
+        rm -rf sccache-v0.5.3-x86_64-unknown-linux-musl*
```

Comment:
this is a year old, why aren't we using the github action?

https://github.com/marketplace/actions/sccache-action